tests/integration_rust/upgrade_tests.rs

```diff
@@ -64,3 +65,86 @@ async fn pypi_dependency_index_preserved_on_upgrade() {
     let redacted_content = content.replace(&Platform::current().to_string(), "[PLATFORM]");
     assert_snapshot!(redacted_content);
 }
+
+#[cfg_attr(not(feature = "slow_integration_tests"), ignore)]
+#[tokio::test]
+async fn upgrade_command_updates_platform_specific_version() {
+    setup_tracing();
+
+    let platform = Platform::current();
+    let pixi = PixiControl::from_manifest(&format!(
+        r#"
+        [workspace]
+        channels = ["https://prefix.dev/conda-forge"]
```

Comment:
Can you check if these tests could be run using the in memory channels or the channels in the `tests/data/channels`?